projects/extension/sql/idempotent/909-text-to-sql-openai.sql

```diff
@@ -460,7 +460,7 @@ begin
         , stop=>_stop
         , temperature=>_temperature
         , top_p=>_top_p
-        , openai_user=>_openai_user
+        --, openai_user=>_openai_user
```

Comment:
Would the fix here just be `client_config => jsonb_build_object('base_url', _base_url, 'openai_user', _openai_user)` vs totally removing these?

Or is it that we need to not pass these values in if `null`?

Reply:
Yes, that would be the best fix, but since this is all changing and we aren't using this feature, I just did the fastest thing possible to get it working again